lib/project-data-updater.js

```diff
@@ -44,9 +44,7 @@ function ProjectDataUpdater(config, repository, updateLock, done) {
 
   var that = this;
   this.done = done || function(err) {
-    if (err) {
-      console.log(err);
-    } else {
+    if (err === null) {
```

Comment:
The other option would be to pass a simple object that looks something like

{
  error: {
    message: "some message goes here"
  }
}

... but both `message` and `error` are optional; if there is no `error`
then it assumes success; likewise it only prints an error message if there
is a `message.`

So essentially, and empty object (rather than a success/error boolean)
would mean success.

On Fri, Nov 13, 2015 at 10:48 AM, Mike Bland <notifications@github.com>
wrote:

> In lib/project-data-updater.js
> <https://github.com/18F/team-api-server/pull/1#discussion_r44796515>:
>
> > @@ -44,9 +44,7 @@ function ProjectDataUpdater(config, repository, updateLock, done) {
> >
> >    var that = this;
> >    this.done = done || function(err) {
> > -    if (err) {
> > -      console.log(err);
> > -    } else {
> > +    if (err === null) {
>
> Yeah, so per my later comment, we can have function(err, noUpdate) (or
> some other parameter name) here. noUpdate will only be defined as true if
> the webhook didn't trigger an update, so checking for noUpdate === true
> will help differentiate between "updated without error" and "no update, no
> error".
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/18F/team-api-server/pull/1/files#r44796515>.
>